src/components/LessonGroups/index.js

```diff
@@ -25,36 +11,41 @@ export default ({instructor}) => {
 
   const items = compact([
     {
-      title: inProgressTitleText,
+      title: <Text message='lessonGroups.inProgress.title' />,
       states: [
         'accepted',
         'claimed',
         'rejected',
       ],
-      fallbackDescription: noInProgressLessonsDescriptionText,
       includeLessonsInCourses: true,
     },
     {
-      title: inReviewTitleText,
-      description: inReviewDescriptionText
-        + (instructor && hasUnlockedSelfReview(instructor.published_lessons)
-          ? ` ${selfReviewDescriptionText}`
-          : ''),
+      title: <Text message='lessonGroups.inReview.title' />,
+      description: (
+        <span>
+          <Text message='lessonGroups.inReview.description' />
+          {instructor && hasUnlockedSelfReview(instructor.published_lessons)
+            ? <span>
+                <span>{` `}</span>
+                <Text message='lessonGroups.inReview.selfApproval' />
+              </span>
+            : null
+          }
+        </span>
+      ),
```

Comment:
Concatenates a longer description if the instructor can self-approve.
![image](https://cloud.githubusercontent.com/assets/5497885/24312598/ecafcfce-109e-11e7-88e8-052dfd70b160.png)